HISTORY.rst

```diff
@@ -2,6 +2,12 @@
 History
 =======
 
+0.4.0
+-----
+
```

Comment:
```suggestion
This is an interim version making one step toward semi-distributed modeling support. Model configuration is still in flux and will be significantly modified with 0.5. The major change in this version is that model configuration requires the creation of HRU objects, instead of simply passing area, latitude, longitude and elevation. 
```

Reply:
Actually we still have an adapter to support the old HRU quatuor.. :violin: :musical_keyboard:

Reply:
adjusted